server/webapp/WEB-INF/rails.new/app/views/new_navigation_elements/_footer.html.erb

```diff
@@ -22,4 +22,9 @@
             </span>
       </div>
   </div>
+  <script type="application/javascript">
+    require(['models/shared/version_updater'], function(VersionUpdater) {
```

Comment:
See [the production build](https://build.gocd.io/go/tab/build/detail/installers/241/dist/1/dist) does not package this file. So this will fail in production. Additionally even if you do package the file in production, you're likely not going to get the correct hash for `version_updater` in production. Post that, this will cause additional http requests to the server. Making for a less-than-ideal page load experience. Any possibility that we add this to each SPA?

We already require `foundation.dropdownMenu` and a few other friends for the navigation header on top, so don't mind putting that in the top level require itself